src/create-storage.js

```diff
@@ -11,7 +11,7 @@ let isWorkerAvailable = window.Worker;
 
 let workify, worker;
 if (isWorkerAvailable) {
-	workify = require('webworkify');
+	workify = isWebpack ? require('webworkify-webpack') : require('webworkify');
```

Comment:
Is there a better way to do replacement, like in `browser` field in `package.json`?
Or mb webpack has it's own replacement declarations?